science/stellarium/Portfile

```diff
@@ -18,12 +18,13 @@ long_description \
     set your coordinates and go.
 homepage        http://stellarium.org/
 
-master_sites    sourceforge
-
-version         0.16.1
+version         0.18.1
 revision        1
-checksums       rmd160 96fcc1e87ebdffeb5c30afc899bbb3289ac88f36 \
-                sha256 b001c252afddaa1c470fb444fcbb5595b7f7a7db028d884743d20bcb5635fd20
+
+master_sites    https://github.com/Stellarium/stellarium/releases/download/v${version}/
```

Comment:
You should probably be using the Github portgroup, so that it is not necessary to specify the master_sites, distfiles, etc. There's documentation in the MacPorts guide.